### ISSUE TITLE
Configuration file

### ISSUE THREAD

**Issue by tfius** [fairDataSociety/fairOS-dfs#156]
Add configuration file that is read on start:
- CORS settings
- postage stamp Id 

any other ideas ?

**Comment by asabya** [comment]
adding these for now
```
bee:
  bee-api-endpoint: 127.0.0.1:1633
  bee-debug-api-endpoint: 127.0.0.1:1635
  postage-batch-id: someId
cookie-domain: localhost
cors-allowed-origins: [localhost:8080]
dfs:
  config: /home/User/.dfs.yaml
  data-dir: /home/User/.fairOs/dfs
  ports:
    http-port: :9090
    pprof-port: :9091
verbosity: info
```

**Comment by tfius** [comment]
looks good for now

**Comment by asabya** [comment]
#160 will deprecate two flags `--beeHost` & `--beePort`, instead it will add two new flags 
 - `--beeApi` full api enpoint url 
 - `--beeDebugApi` full debug api endpoint for stamps api
 
 it will create a `.dfs.yaml` at user home directory or user defined path passed via `--config` flag.
 
 it will also add a command `config` that will print current config

**Comment by asabya** [comment]
#103 this can be added in the config file. @IgorShadurin any suggestions?

**Comment by asabya** [comment]
#103 was not added. although config changed got released in v0.7.3
